src/submission/validation-clinical/donor.ts

```diff
@@ -138,81 +145,73 @@ const crossFileValidator = async (
         ),
       );
     } else {
-      const lostToFollowUpDiagnosisId =
-        donorClinicalEventIdMatch?.clinicalInfo?.submitter_primary_diagnosis_id;
-
-      const lostToFollowUpInterval =
-        (typeof donorClinicalEventIdMatch?.clinicalInfo?.interval_of_followup === 'number' &&
-          donorClinicalEventIdMatch.clinicalInfo.interval_of_followup) ||
-        0;
-
-      const invalidTreatmentIntervals = treatments.filter(treatment => {
-        const treatmentRecord = treatment.clinicalInfo;
-
-        const { treatment_start_interval, treatment_duration } = treatmentRecord;
-
-        const treatmentStartInterval =
-          typeof treatment_start_interval === 'number' ? treatment_start_interval : 0;
-
-        const treatmentDuration = typeof treatment_duration === 'number' ? treatment_duration : 0;
-
-        const totalTreatmentTime = treatmentStartInterval + treatmentDuration;
-
-        return totalTreatmentTime > lostToFollowUpInterval;
+      const lostToFollowUpClinicalInfo = donorClinicalEventIdMatch.clinicalInfo;
+
+      const lostToFollowUpStartInterval =
+        Number(
+          lostToFollowUpClinicalInfo.interval_of_followup ||
+            lostToFollowUpClinicalInfo.treatment_start_interval ||
+            lostToFollowUpClinicalInfo.specimen_acquisition_interval,
```

Comment:
Need to check across different keys for different entities; was previously only checking `interval_of_followup`